### ISSUE TITLE
Evaluate napari-aicsimageio

### ISSUE THREAD

**Issue by dpshelio** [HealthBioscienceIDEAS/microscopy-novice#11]
https://github.com/AllenCellModeling/napari-aicsimageio

## Definition of done

Fit for purpose on teaching
- [x] metadata visualisation
- [ ] Open file formats required
- [x] Easy to install and use (e.g., on M1/2 chips)

**Comment by p-j-smith** [comment]
I had a look at `napari-aicimageio` this morning and it seems it could be useful for visualising lots of different data formats and their metadata.

## Installation
```
mamba create -n napari -c conda-forge python=3.10 napari bioformats_jar "PySide6<6.5"
mamba activate napari
python -m pip install napari-aicsimageio
```
I haven't got an M1/2 Mac, but the above worked on my Intel Mac and [should work on M1 Macs too](https://github.com/AllenCellModeling/napari-aicsimageio/issues/74#issuecomment-1531914160).

## File formats required

I'm not sure what file formats are needed for the workshop, but the plugin supports reading a widge range of formats. 

However, it does not support writing _any_ format. So the plugin is currently useful for visualising images and their metadata, not doing analysis within napai and exporting the results. The [underlying library](https://github.com/AllenCellModeling/aicsimageio) has got [writing capabilities](https://allencellmodeling.github.io/aicsimageio/aicsimageio.writers.html), and there's an [old issue to add writing support to the plugin](https://github.com/AllenCellModeling/napari-aicsimageio/issues/6). We could consider opening a pr to add writing support to the plugin, at least for file formats we need for the workshop.

## Metadata visualisation

It's straightforward to view the image metadata using the plugin:

1. Open the plugin: `Plugins > OME Tree Widget (ome-types)`
2. Load an image using the plugin: `File > Open with plugin > File(s)`

The metadata is then displayed in the plugin widget.

## Other thoughts

The plugin is developed by the same team that developers the `aicimageio`, the underlying library. However, the plugin hasn't been updated in over a year even though there have been several updates to `aicimageio`. The plugin still works for now, but we should encourage the workshop participants to consider alternatives in the future if this one goes unmaintained.

**Comment by davecash75** [comment]
HI @p-j-smith  this was the comment from one of the microscopy experts on this point "Vital to include various data formats and limitations e.g. Tiff (BigTiff) , jpeg, compression types (lossless etc). Converting between types. Proprietary formats and Bio-formats" I have written to them asking if there are any examples he could provide. Are there also any example data in addition to the ones @K-Meech already found in the napari examples that we can start to use here?

**Comment by MonikaSvata** [comment]
- Check the licences
- Consider using histology data

**Comment by davecash75** [comment]
As mentioned in #5, Omero has some sample data at https://downloads.openmicroscopy.org/images/

**Comment by MonikaSvata** [comment]
- EICS testing data will not be used due to licencing requirements. 
- We won't consider histology data for opening in Napari. 

- Tasks left: 
Testing with data from Nick (still waiting for the data).

**Comment by p-j-smith** [comment]
For the resources mentioned in https://github.com/AllenCellModeling/aicsimageio/discussions/544#discussioncomment-7568092:

- [Cell Feature Explorer](https://cfe.allencell.org/) and the [Allen Cell Quilt Package](https://open.quiltdata.com/b/allencell/packages/) both require a Python package ([quilt](https://docs.quiltdata.com/walkthrough/getting-data-from-a-package)) to be able to download the data, e.g.

```python
import quilt3
pkg = quilt3.Package.browse("aics/hipsc_single_cell_image_dataset", "s3://allencell")
pkg.fetch("</path/to/image/>")
```

This might be too much to ask of the participants, especially if we're not requiring Python experience.

- [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/) - there are some `lif` and `czi` datasets that could be of use. However, the `napari-aicsimageio` plugin and shows metadata for OME-TIFF files.

I think something like the [MitoCheck OME-TIFF dataset](https://docs.openmicroscopy.org/ome-model/5.6.3/ome-tiff/data.html#mitocheck) would be good - it's relatively small (126 MB), contains metadata that we can view in napari, and is a 2D timeseries so could potentially be used for both cell counting and/or cell tracking

**Comment by K-Meech** [comment]
So I took a look at the MitoCheck OME-TIFF dataset - looks really nice + easy to download! It's a nice example of a time series, so I'm using it in the multi-dimensional images episode https://github.com/HealthBioscienceIDEAS/microscopy-novice/pull/31 . Unfortunately it doesn't have a pixel size set (e.g. in microns), so it's not the best example for the 'filetypes and metadata' episode where I want to highlight this.  

As an alternative - I was thinking of [this yeast dataset](https://downloads.openmicroscopy.org/images/Zeiss-CZI/idr0011/Plate1-Blue-A_TS-Stinger/) from the OME example data (the top one 'Plate1-Blue-A-12-Scene-3-P3-F2-03.czi'). This is small, easy to download and has a lot of metadata that can be browsed in aicsimagio, including pixel size. Also, it's released under a CC 4.0 license as stated in the readme one level up.
